.circleci/config.yml

```
@@ -335,14 +335,37 @@ jobs:
             php --ini
             php --version
 
+  test-redis:
+    docker:
+      - image: ghcr.io/lecaoquochung/docker-images:redis-latest
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: |
+            whoami;pwd;ls -all; 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)